### PR TITLE
Tbozhikov/how ns work article

### PR DESCRIPTION
**Pull request by tbozhikov**
- add new page How NativeScript Works
- remove common diagram from Welcome page

### CODE REVIEW

File: docs/start/how-it-works.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+## Runtimes
+
+The runtimes enable you to call APIs in the Android and iOS frameworks using JavaScript code. To do that they use JavaScript Virtual Machines - Google’s V8 for Android and WebKit’s JavaScriptCore implementation distributed with iOS 7.0+. Find more information in [Android Runtime Overview](../runtimes/android/overview.md) and [iOS Runtime Overview](../runtimes/ios/Overview.md).
+
+## Core Modules
+
+The Core Modules are there to provide the abstractions needed to access the underlying native platforms. Take the Gestures module for example. It defines a common JS API that when called from the application TypeScript/JavaScript translates to native gestures APIs calls (thanks to the Runtimes). 
+
+Another thing the Core Modules provide is the "original" NativeScript way for defining UI, data-binding, navigation. Along with it, Angular and Vue can be used as Application Framework technologies. Read more in the [Modules Documentation](../core-concepts/modules.md).
```
Comment (etabakov) [inline docs/start/how-it-works.md]: I think the official naming of Vue is "Vue.js".

File: docs/start/how-it-works.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+---
+title: How NativeScript Works
+description: NativeScript - technology stack diagram.
+position: 10
+publish: true
+slug: how-it-works
+previous_url: /index
```
Comment (etabakov) [inline docs/start/how-it-works.md]: The previous_url is not required

File: docs/start/how-it-works.md (hunk 1 above)
Comment (etabakov) [inline docs/start/how-it-works.md]: Not sure that "original" is clear enough. Can we replace it with "a basic, xml-based way for defining UI ..."?

**Review by etabakov — COMMENTED** [review]
Looks great! Some minor comments ...

File: docs/start/how-it-works.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
+---
+title: How NativeScript Works
+description: NativeScript - technology stack diagram.
```
Comment (tjvantoll) [inline docs/start/how-it-works.md]: Well this is more than a diagram. Maybe...

A broad overview of the technologies that make NativeScript work

File: docs/start/how-it-works.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+## NativeScript CLI
+
+It is the command-line interface that lets you create, build, and run apps using NativeScript. Runs on Windows, macOS or Linux. Find more information in the [NativeScript CLI repo](https://github.com/NativeScript/nativescript-cli).
+
+## NativeScript Plugins
+
+The NativeScript plugins are building blocks that encapsulate some functionality and help developers build apps faster (just like the NativeScript Core Modules, which is a plugin). Most are community-built, written in TypeScript/JavaScript. Some can include native libraries, which are called from the TS/JS code thanks to the Runtimes. More information in the [Plugins documentation](../plugins/plugins.md).
+
+
```
Comment (tjvantoll) [inline docs/start/how-it-works.md]: As a suggestion, you might want to link to https://developer.telerik.com/featured/nativescript-works/ as further reading at the end.

File: docs/start/how-it-works.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+![NativeScript common diagram](../img/ns-common.png)
+
+## Runtimes
+
+The runtimes enable you to call APIs in the Android and iOS frameworks using JavaScript code. To do that they use JavaScript Virtual Machines - Google’s V8 for Android and WebKit’s JavaScriptCore implementation distributed with iOS 7.0+. Find more information in [Android Runtime Overview](../runtimes/android/overview.md) and [iOS Runtime Overview](../runtimes/ios/Overview.md).
+
+## Core Modules
+
+The Core Modules are there to provide the abstractions needed to access the underlying native platforms. Take the Gestures module for example. It defines a common JS API that when called from the application TypeScript/JavaScript translates to native gestures APIs calls (thanks to the Runtimes).
```
Comment (tjvantoll) [inline docs/start/how-it-works.md]: It defines a common JS API that translates application TypeScript/JavaScript code into native gestures APIs calls (thanks to the Runtimes).

File: docs/start/how-it-works.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+
+## Runtimes
+
+The runtimes enable you to call APIs in the Android and iOS frameworks using JavaScript code. To do that they use JavaScript Virtual Machines - Google’s V8 for Android and WebKit’s JavaScriptCore implementation distributed with iOS 7.0+. Find more information in [Android Runtime Overview](../runtimes/android/overview.md) and [iOS Runtime Overview](../runtimes/ios/Overview.md).
+
+## Core Modules
+
+The Core Modules are there to provide the abstractions needed to access the underlying native platforms. Take the Gestures module for example. It defines a common JS API that when called from the application TypeScript/JavaScript translates to native gestures APIs calls (thanks to the Runtimes). 
+
+Another thing the Core Modules provide is a basic XML-based way for defining UI, data-binding, navigation. Along with it, Angular and Vue.js can be used as Application Framework technologies. Read more in the [Modules documentation](../core-concepts/modules.md).
```
Comment (tjvantoll) [inline docs/start/how-it-works.md]: data-binding, and navigation

File: docs/start/how-it-works.md [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+
+## NativeScript CLI
+
+It is the command-line interface that lets you create, build, and run apps using NativeScript. Runs on Windows, macOS or Linux. Find more information in the [NativeScript CLI repo](https://github.com/NativeScript/nativescript-cli).
```
Comment (tjvantoll) [inline docs/start/how-it-works.md]: The CLI runs on Windows, macOS or Linux, and you can find more information...

File: docs/start/how-it-works.md [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+
+## NativeScript Plugins
+
+The NativeScript plugins are building blocks that encapsulate some functionality and help developers build apps faster (just like the NativeScript Core Modules, which is a plugin). Most are community-built, written in TypeScript/JavaScript. Some can include native libraries, which are called from the TS/JS code thanks to the Runtimes. More information in the [Plugins documentation](../plugins/plugins.md).
```
Comment (tjvantoll) [inline docs/start/how-it-works.md]: You can find more information in the...

**Review by tjvantoll — APPROVED** [review]
I only had a few minor grammatical things. I like the new page a lot and I think this is a good place for this content.

**Pull request merged**